README.md

```diff
@@ -1,6 +1,6 @@
 # balena CLI
 
-> The official balena CLI tool.
```

Comment:
This change was proposed by @xginn8 ([here](https://github.com/balena-io/balena-cli/pull/1181/files#diff-04c6e90faac2675aa89e2176d2eec7d8R4)). It may originally have been intentional to format it with indentation -- I don't mind. :-)

Reply:
According to the [official github markdown docs](https://guides.github.com/features/mastering-markdown/), `>` is used for quoting, which feels wrong to me for this subtitle.